src/views/EditorHome.vue

```diff
@@ -544,6 +544,36 @@ onMounted(async () => {
     store.state.acceptTerms !== "Accepted";
 
   isCompletedInitialStartup.value = true;
+
+  // 代替ポートをトースト通知する
+  if (!store.state.confirmedTips.noticeAltPortInfo) return;
+  const altPortInfo = await store.dispatch("GET_ALT_PORT_INFO");
```

Comment:
（ちょっと長いです）

上の`GET_ENGINE_INFOS`みたく、stateに`AltPortInfo`を持っておいてそこにSETするようにすると、dispatchせずにaltPortInfoをwatchしたりreactiveにしたりできそうです。

例えば今だとたぶん最初のEditorHome作成時に１回トーストが表示されますが、エンジン再起動時は表示されない感じになっていると思います。
これをstate.altPortInfoをwatchする形にすればまた表示することができる･･･など！

ただまあだいぶややこしくなるので、別に良いかなとちょっと思いました！！
もしVueに興味があれば挑戦してみるとちょっと面白いかもです。
（でももしほかのissueなどの挑戦を考えられているなら、そちらをぜひお願いしたいです！！）

とりあえずFIXMEを書いてみるのとかどうでしょう。
```suggestion
  // 代替ポートをトースト通知する
  // FIXME: トーストが何度も出るようにする（altPortInfoをstateに持たせてwatchする）
  if (!store.state.confirmedTips.noticeAltPortInfo) return;
  const altPortInfo = await store.dispatch("GET_ALT_PORT_INFO");
```

Reply:
確かにそうですね...！　まだVuexが不慣れなので, FIXME を残しておき,  
上の https://github.com/VOICEVOX/voicevox/pull/1267#discussion_r1174612000 と合わせてリファクタリングのPRを送りたいと思います！